Gemfile

```diff
@@ -11,3 +11,9 @@ end
 group :development, :test do
   gem 'rubocop', '1.20'
 end
+
```

Comment:
Review doc suggests to group gems - so group together the ones related to testing, e.g. Rspec, capybara,